source/common/http/alternate_protocols_cache_manager_impl.cc

```diff
@@ -53,6 +53,20 @@ AlternateProtocolsCacheSharedPtr AlternateProtocolsCacheManagerImpl::getCache(
 
   AlternateProtocolsCacheSharedPtr new_cache = std::make_shared<AlternateProtocolsCacheImpl>(
       dispatcher.timeSource(), std::move(store), options.max_entries().value());
+
+  for (const envoy::config::core::v3::AlternateProtocolsCacheEntry& entry :
+       options.prepopulated_entries()) {
+    const AlternateProtocolsCacheImpl::Origin origin = {"https", entry.hostname(), entry.port()};
+    std::vector<AlternateProtocolsCacheImpl::AlternateProtocol> protocol = {
+        {"h3", entry.hostname(), entry.port(),
+         dispatcher.timeSource().monotonicTime() + std::chrono::days(7)}};
```

Comment:
Is 7 days a standard value here? Maybe we should explicitly name this in the docs? It's implict in the `ma` in the example, but might not be obvious

This also seems to mean that if someone pushed config to pre-fill this, then reverted the config, we'd retain a cache entry for that for 7 days. This is probably fine but maybe worth calling out so nobody gets confused about quic being used for a connection that isn't prefilled with the current config?